src/main/java/muramasa/antimatter/AntimatterAPI.java

```diff
@@ -187,33 +187,10 @@ public static void registerFluidCell(ItemStack stack) {
         return cells;
     }
 
-    /** Cover Registry Section **/
-    private final static Object2ObjectMap<String, Cover> COVER_REGISTRY = new Object2ObjectOpenHashMap<>();
-    private final static Object2ObjectMap<Item, Cover> CATALYST_TO_COVER = new Object2ObjectOpenHashMap<>();
-
-    /**
-     * Registers a cover behaviour. This must be done during preInit.
-     * @param cover The behaviour instance to be attached.
-     */
-    public static void registerCover(Cover cover) {
-        cover.onRegister();
-        COVER_REGISTRY.put(cover.getId(), cover);
-    }
-
-    public static Cover getCover(String name) {
-        return COVER_REGISTRY.get(name);
-    }
-
-    public static void registerCoverStack(ItemStack stack, Cover cover) {
-        CATALYST_TO_COVER.put(stack.getItem(), cover);
-    }
-
-    public static Cover getCoverFromCatalyst(ItemStack stack) {
-        return CATALYST_TO_COVER.get(stack.getItem());
-    }
+    public static Cover getCover(String name) { return get(Cover.class, name); }
```

Comment:
Just call AntimatterAPI#get(Cover.class, id) where you need it, we don't need a method in AntimatterAPI for it.

Reply:
I'm not even sure if this function has any use, left it there just in case

Reply:
Yeah, I just didn't want a random get helper in AntimatterAPI, for the time being sure.